src/fn/initialise-project-layout.js

```diff
@@ -9,14 +9,14 @@ const LAYOUT = {
   fields: [],
   cards: [],
   context: {}
-};`,
+};\n`,
```

Comment:
FYI, I ended up adding these new lines in here after getting failures in the e2e tests.  Basically the default files we were writing here were not compatible with the `@medic/eslint-config` rules (because they did not have an empty line at the end.  This caused linting failures when running the e2e tests because the e2e test project inherits the config from this repo's .eslintrc file.  Eventually we may need to make some changes so this inheritance does not happen, but in this case it seemed appropriate to clean up these files here.